### DISCUSSION TITLE
Welcome to the DevOps Track!

### DISCUSSION THREAD

**Discussion by AndreaGriffiths11** [github/india#6] [Satellite India 2021 Q&A]
Increase efficiency, automate workflows and create secure software all at once.

**Comment by dhruvg20** [comment]
Good Morning :pray:  🙇🏽  

Current session going on - [What's new with GitHub Actions](https://githubsatellite.com/sessions/whats-new-with-github-actions/#) - Helpful Links:

1. [GitHub Actions Documentations](https://docs.github.com/en/actions)
2. [GitHub Marketplace - Actions](https://github.com/marketplace?type=actions)

Please feel free to ask any questions or inputs.

**Reply by dhruvg20** [reply]
- [Creating a Docker container action](https://docs.github.com/en/actions/creating-actions/creating-a-docker-container-action)

**Reply by dhruvg20** [reply]
[GitHub Action for deploying to Azure Web App](https://github.com/marketplace/actions/azure-webapp)

**Reply by dhruvg20** [reply]
GitHub Actions [Events that trigger workflows](https://docs.github.com/en/actions/reference/events-that-trigger-workflows)
GitHub [Encrypted secrets](https://docs.github.com/en/actions/reference/encrypted-secrets)

**Comment by dhruvg20** [comment]
We start the next session on - [From right to left: Security and secure development](https://githubsatellite.com/sessions/from-right-to-left/) ❤️  🙇🏽

**Reply by dhruvg20** [reply]
[Code scanning is now available!](https://github.blog/2020-09-30-code-scanning-is-now-available/)
[About code scanning](https://docs.github.com/en/code-security/secure-coding/about-code-scanning)

**Comment by dhruvg20** [comment]
We starting the next session - [How Linkerd transforms code into running software](https://githubsatellite.com/sessions/how-linkerd-transforms-code-into-running-software/)

**Reply by dhruvg20** [reply]
As shared by Tarun [Caching dependencies to speed up workflows](https://docs.github.com/en/actions/guides/caching-dependencies-to-speed-up-workflows)

**Reply by dhruvg20** [reply]
Tarun shared about [What are annotations?](https://github.community/t/what-are-annotations/16173)

**Reply by dhruvg20** [reply]
[GitHub Packages](https://docs.github.com/en/packages)
[GitHub Codespaces](https://docs.github.com/en/github/developing-online-with-codespaces/about-codespaces)

**Comment by dhruvg20** [comment]
Starting the next session - [Building event driven apps on Kubernetes with Fission](https://githubsatellite.com/sessions/building-event-driven-apps/) 🎉
